src/modules/Staff/Service.php

```diff
@@ -695,4 +695,25 @@ public function authorizeAdmin($email, $plainTextPassword)
 
         return $this->di['auth']->authorizeUser($model, $plainTextPassword);
     }
+
+    public function sendPasswordResetConfirmation($hash)
+    {
+        $reset = $this->di['db']->findOne('AdminPasswordReset', 'hash = ?', [$hash]);
+        if (!$reset instanceof \Model_AdminPasswordReset) {
+            throw new \Box_Exception('The link have expired or you have already confirmed password reset.');
+        }
+        $new_pass = $this->di['tools']->generatePassword();
+        $c = $this->di['db']->getExistingModelById('Admin', $reset->admin_id, 'User not found');
+        $c->pass = $this->di['password']->hashIt($new_pass);
+        $this->di['db']->store($c);
+
+        // send email
+        $email = [];
+        $email['to_client'] = $reset->admin_id;
```

Comment:
Same here  we can't just send "Admin id " one. 

We need to rewrite the email class so it also support admin users

Reply:
Should be to_staff instead of "to_client"

Reply:
`to_client` allows you to send to one client but `to_staff` goes to all staff? I could just use the to option?

Reply:
Yes

Reply:
wouldnt work. will have to add a `to_admin` Email\Service

Reply:
throw new \Box_Exception('Receiver is not defined. Define to or to_client or to_staff parameter');

Is available It should work 

```
            $staffService = $this->di['mod_service']('staff');
            $staff = $staffService->getList(['status' => 'active', 'no_cron' => true]);
            $vars['staff'] = $staff['list'][0];
```

Reply:
It only send to the first one 

 $vars['staff'] = $staff['list'][0];

I like the software we should add support for providing staff id at least